src/Main.java

```diff
@@ -6,13 +6,13 @@ public static void main(String[] args) {
         int anoDeLancamento = 2022;
         System.out.println("O ano de lançamento do filme é: " + anoDeLancamento);
         boolean incluidoNoPlano = false;
-        double notaDoFilme = 9.2;
-        String sinopse = "Filme de aventura relacionado a aviação";
+        double media  = (7.2 + 6.9 + 9 +8.2) / 4;
+        int estrelas = (int) media / 2;
+                String sinopse = "Filme de aventura relacionado a aviação";
```

Comment:
Precisa arrumar o espaçamento

Reply:
Deixar os prints mais para baixo?

Reply:
Talvez fosse uma boa, mas to falando do `sinopse` mesmo que ta desalinhado